docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -132,6 +132,14 @@ The remote control framework also allows mobile applications to send simulated b
 ## Integration
 For remote control to work, the head unit must support SDL Core v.4.4 or newer. Also your app's @![iOS]`appType`!@ @![android, javaSE, javaEE]`appHMIType`!@ must be set to `REMOTE_CONTROL`.
 
+### Pre Core v6.0
```

Comment:
Change this section to a note stating "When connected to pre-Core 6.0 systems only the main module of a module type can be controlled. The modules you may control will depend on the OEM."

Please spell-check before submitting updates as well.

Reply:
changed